proj/scala-rewrites.conf

```diff
@@ -0,0 +1,36 @@
+// https://github.com/scala/scala-rewrites.git#master
+
+// This project is unusual in that we build it using only binary
+// dependencies, since the community build's scalafix version is too
+// old and we can't upgrade it because it has too many dependencies.
+// (And scala-rewrites is 2.12-only because it lives in
+// sbt-plugin-land.)
+
+// This is trickier-than-normal because semanticdb-scalac and
+// scalafix-testkit are CrossVersion.full, so we have to override that
+// to use older binaries, since binaries don't exist for Scala nightlies.
```

Comment:
Perhaps we can get dbuild / the community build version the scala as `-bin-...`?

Reply:
not sure what you mean

Reply:
Ah, wait (_he says to himself_) it's `CrossVersion.patch` that would work better here, right?

Reply:
still not sure what you mean

Reply:
OK, sorry.  What I'm trying to say is how do we avoid having to do the `removeDependency` and `set input / libraryDependencies += ...` in the community build?  I think the answer is `CrossVersion.patch`, but I reckoned you would know best.

Reply:
`CrossVersion` strips off the `-bin-` suffix, but we'd still be left with e.g. 2.12.12, which doesn't exist yet

Reply:
Ah, right.  Damn.